src/Helper/Navigation/AbstractHelper.php

```diff
@@ -432,14 +434,15 @@ public function htmlify(AbstractPage $page)
     protected function translate($message, $textDomain = null)
     {
         if (! is_string($message) || empty($message)) {
-            return $message;
+            return '';
```

Comment:
BC issue?

Reply:
Yes, you're right of course - I've reverted and baselined the errors. Thanks for the review @Ocramius